nuxeo-mongodb-audit/README.md

```diff
@@ -0,0 +1,29 @@
+nuxeo-mongodb-audit
+===================
+
+## About
+
+This project provides a backend based on MongoDB for Nuxeo for Audit Service.
+
+The idea is to use MongoDB as storage backend for the Audit trail entries.
+
+## How it works
+
+If you install the package marketplace, you just need to activate mongo-audit template to make it works.
+
+Then an MongoDB based `AuditBackend` is contributed at startup to replace the default JPA based one.
+
+The backend use the `audit` MongoDB connection or the `default` one if not.
+Entries are stored in `audit` collection unless if you override it by contributing to ConfigurationService like this:
+
+    <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+      <property name="nuxeo.mongodb.audit.collection.name">myCollection</property>
+    </extension>
+
+The queries and PageProviders are based on MongoDB native DSL : no automatic conversion is done between EJBQL and MongoDB DSL.
```

Comment:
I would remove "no automatic conversion is done between EJBQL and MongoDB DSL" since the JPA back-end is not the default one.